src/models/server.cpp

```diff
@@ -191,27 +194,17 @@ uint32_t Server::choosePort() const {
   }
   Q_ASSERT(length < 65536);
 
-  while (true) {
-    // Pick a port at random.
-    quint32 r = QRandomGenerator::global()->generate() % length;
-    quint32 port = 0;
-
-    for (const QPair<uint32_t, uint32_t>& range : m_portRanges) {
-      if (r <= (range.second - range.first)) {
-        port = r + range.first;
-        break;
-      }
-      r -= (range.second - range.first + 1);
-    }
-    Q_ASSERT(port != 0);
+  // Pick a port at random.
+  quint32 r = QRandomGenerator::global()->generate() % length;
```

Comment:
Could this end up generating a port 53, or cause the resulting range to be 53?  Is that still an issue with Mullvad blocking it?

Reply:
Mullvad no longer blocks port 53, and we rely on that behaviour to support custom DNS servers.